dandi/consts.py

```diff
@@ -134,3 +134,6 @@
 #: HTTP response status codes that should always be retried (until we run out
 #: of retries)
 RETRY_STATUSES = (500, 502, 503, 504)
+
+external_file_extensions = [".mp4", ".avi", ".wmv", ".mov", ".flv"]
```

Comment:
Constant names should be in all-caps.

Reply:
we should then be following this for all the constants defined above also which can go in another PR